pkgs/applications/networking/cluster/terraform-providers/lxd/default.nix

```diff
@@ -0,0 +1,26 @@
+# This file was generated by https://github.com/kamilchm/go2nix v1.3.0
+{ stdenv, buildGoPackage, fetchgit }:
+
+buildGoPackage rec {
+  name = "terraform-provider-lxd-unstable-${version}";
+  version = "2019-04-03";
+  rev = "ed5c4247171d35f012c765c585cd24a44e4f234e";
+
+  goPackagePath = "github.com/sl1pm4t/terraform-provider-lxd";
+
+  src = fetchgit {
```

Comment:
You can use `fetchFromGitHub` instead

Reply:
This derivation was auto generated with go2nix. I thought it better not to modify auto generated code